src/classloader/opcode_parser.rs

```diff
@@ -64,16 +65,22 @@ fn parse_ldc<'a>(
         RuntimeCPEntry::FloatInfo(value) => {
             Ok((current_content, OpCode::Ldc(Ldc::Float(*value))))
         },
+        RuntimeCPEntry::LongInfo(value) => {
+            Ok((current_content, OpCode::Ldc(Ldc::Long(*value))))
+        },
+        RuntimeCPEntry::DoubleInfo(value) => {
+            Ok((current_content, OpCode::Ldc(Ldc::Double(*value))))
+        },
         _ => panic!("{:?} Unsupported Type for Ldc ", cp_entry),
```

Comment:
For other panics you changed the message from "unsupported" to "invalid" in this PR? Did you consider to change this one too? Or am I missing an Ldc variant?

Reply:
I am pretty sure, the only time we use "not valid" is in the underscore case, where we changed it from unsupported. We changed it, since we now have a case for every valid Java Bytecode OpCode, so if we run into this case the byte is actually not a valid OpCode from the specification. All the other times we use "unsupported", since it is possible in the specification but not yet implemented by us.